tests/PKSim.Tests/IntegrationTests/DistributedParameterToTableParameterConverterSpecs.cs

```diff
@@ -95,10 +96,19 @@ public override void GlobalContext()
       {
          base.GlobalContext();
          _individual = DomainFactoryForSpecs.CreateStandardIndividual();
-
          _simulation = DomainFactoryForSpecs.CreateModelLessSimulationWith(_individual, _compound, _iv).DowncastTo<IndividualSimulation>();
          _simulation.AllowAging = true;
          DomainFactoryForSpecs.AddModelToSimulation(_simulation);
+         
+      }
+
+      [Observation]
+      public void should_be_able_to_run_the_simulation()
+      {
+         var simulationRunner = IoC.Resolve<ISimulationRunner>();
+         //this config may lead to negative values. We want to check here that the simulation can run without errors
+         simulationRunner.RunSimulation(_simulation, new SimulationRunOptions{CheckForNegativeValues = false}).Wait();
```

Comment:
@Yuri05 We were creating a lot of simulation aging but NEVER running them lol. Here we go. This was red and became green